apps/docs/content/cs/how-to/ebpf-sqlite-io.md

```
@@ -0,0 +1,48 @@
+# SQLite I/O latence (bpftrace)
+
+## Cíl
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)